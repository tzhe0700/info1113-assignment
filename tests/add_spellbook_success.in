ADD SPELLBOOK spellbooks.csv 111111
EXIT

