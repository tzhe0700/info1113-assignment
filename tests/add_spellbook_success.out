Successfully added: The Standard Book of Spells Grade 1 (Miranda Goshawk).

